Makefile

```diff
@@ -1,30 +1,41 @@
 install:
+	@export PR=https://api.github.com/repos/$$TRAVIS_REPO_SLUG/pulls/$$TRAVIS_PULL_REQUEST; \
+	export BRANCH=$$(if [ "$$TRAVIS_PULL_REQUEST" == "false" ]; then echo $$TRAVIS_BRANCH; else echo `curl -s $$PR | jq -r .head.ref`; fi); \
+	echo "TRAVIS_BRANCH=$$TRAVIS_BRANCH, PR=$$PR, BRANCH=$$BRANCH"; \
+	pattern="refactor_"; \
+	if [[ $$BRANCH =~ $$pattern ]]; then \
```

Comment:
Something in here is not correct yet:

https://travis-ci.org/dedis/cothority/builds/145013562#L161

Most probably you should write `"$$BRANCH"` and `"$$pattern"`.